InstallationGuide.md

```diff
@@ -1,50 +1,39 @@
 # Installation Guide
-## Product DFT
-Install from the command line:
 
-docker container run -d --name [conatainer_name] ghcr.io/catenax-ng/tx-dft-backend:[tag]
+It is necessary to inject the environment variables, credentials and URLs that can be found on application.properties file.
+For more details, please refer configuration section from [README.md](README.md)
 
+### RUN SDE backend in ArgoCD 
+ We have helm chart available for ArgoCD deployment. In deployment, if don't specified specific version, the app version/chart version is automatically picked up by ArgoCD and deployed to the environment using Helm charts.
+
+ In values.yaml you can find `default` as value for all required configuration. You need to change all those values as per your need. For reference, please refer configuration example section.
+ 
+ As part of argo CD deployment using helm chart the postgres database dependency will get provide automatic but for EDC, DigitalTwin and Portal you need to provide valid details as per configuration requirement other wise SDE service will get started with default configuration but will not work as expected.
+
+### RUN SDE Backend in k8ts cluster
+#### Prerequisites
+- k8ts cluster/ minikube
+- helm
+- Docker
+
+ In values.yaml you can find `default` as value for all required configuration. You need to change all those values as per your need. For reference, please refer configuration example section.
+
+ helm repo add sde-backend https://github.com/eclipse-tractusx/dft-backend/tree/main/charts
+   
+ helm install release-name eclipse-tractusx/dft-backend
+
+### RUN SDE Backend Locally
+#### Prerequisites
+- JDK18
+- Postgres 13.2
```

Comment:
No one is using postgres 13 anymore: https://app.int.demo.catena-x.net/ are you sure this is right?

Reply:
Removed the postgres and marked to 11.x.x. version.